cli/aeon/decode.go

```diff
@@ -0,0 +1,102 @@
+package aeon
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/tarantool/go-tarantool/v2/datetime"
+	"github.com/tarantool/go-tarantool/v2/decimal"
+	"github.com/tarantool/tt/cli/aeon/pb"
+)
+
+/*
+decodeValue convert a value obtained from protobuf into a value that can be used as an
+argument to Tarantool functions.
+
+Copy from https://github.com/tarantool/aeon/blob/master/aeon/grpc/server/pb/decode.go
+*/
+func decodeValue(val *pb.Value) (any, error) {
```

Comment:
Just to keep the same code style.
```suggestion
// decodeValue convert a value obtained from protobuf into a value that can be used as an
// argument to Tarantool functions.
// 
// Copy from https://github.com/tarantool/aeon/blob/master/aeon/grpc/server/pb/decode.go
func decodeValue(val *pb.Value) (any, error) {
```